charter.html

```diff
@@ -163,8 +163,16 @@ <h2>Scope</h2>
         <p>
             The Working Group will specify new web platform features intended to
             be implemented in browsers or similar user agents. The purpose of
-            these features is to support web advertising and provide users with
-            privacy guarantees with a strong technical basis.
+            these features is to support web advertising without compromising
+            user privacy.  Here "privacy" minimally refers to appropriate
+            process of personal information.  Ways in which new features might
```

Comment:
processing

Reply:
I think "processing of personal data" is a term of art: https://gdpr-info.eu/art-5-gdpr/

I'm not familiar with "Appropriate process of personal information"